jdbc/src/java/org/apache/hive/jdbc/Utils.java

```diff
@@ -618,7 +618,7 @@ public static JdbcConnectionParams extractURLComponents(String uri, Properties i
             port = Integer.parseInt(Utils.DEFAULT_PORT);
           }
           connParams.setHost(jdbcBaseURI.getHost());
```

Comment:
can we use ``host`` variable here to match the code style

Reply:
> can we use `host` variable here to match the code style

sure, changed